lib/linguist/languages.yml

```
@@ -8195,6 +8195,16 @@ Zimpl:
   tm_scope: none
   ace_mode: text
   language_id: 411
+crontab:
+  type: data
```

The colour I [suggested](https://github.com/github-linguist/linguist/pull/6759#issuecomment-2002443424) earlier:
```suggestion
crontab:
  type: data
  color: "#ead7ac"
```